scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -1620,6 +1622,28 @@ function initialize_pxb_commands()
         disver="--no-version-check"
     fi
 
+    # Check if xtrabackup (PXB) Pro version is used (supports --lock-ddl=REDUCED)
+    if ${pxb_bin_path} --version 2>&1 | grep -q -- '-pro'; then
+        if [[ -z "$lock_ddl_opt" ]]; then
+            lock_ddl="--lock-ddl=REDUCED"
+            wsrep_log_info "PXB Pro detected; using default --lock-ddl=REDUCED for SST"
+        else
+            lock_ddl="--lock-ddl=$lock_ddl_opt"
+        fi    
+    else
+        # If REDUCED is requested but not supported, exit with error
+        if [[ "$lock_ddl_opt" == "REDUCED" ]]; then
```

Comment:
As we discussed on Slack, what if the user sets `reduced` or `ReDuCeD`? In other words, are option values case sensitive or insensitive?